bundles/org.openhab.binding.mqtt.homie/src/main/resources/OH-INF/thing/homie-channels.xml

```diff
@@ -7,11 +7,19 @@
 	<channel-type id="homie-boolean">
 		<item-type>Switch</item-type>
 		<label>Boolean</label>
+		<tags>
```

Comment:
I wouldn't tag this. There's zero semantic information coming from Homie on what this is. It could be read only, and often does _not_ represent power state.